index.html

```diff
@@ -9,12 +9,9 @@
     <link rel='apple-touch-icon' sizes='180x180'
           href='/static/icons/favicon-apple-180x180.png'>
     <link rel="mask-icon" href="/static/icons/mask-icon.svg" color="#3fbbf4">
-    {% if not dev_mode %}
-      <link rel='preload' href='/home-assistant-polymer/build/core.js' as='script'/>
-      {% for panel in panels.values() -%}
-        <link rel='prefetch' href='{{ panel.webcomponent_url }}'>
-      {% endfor -%}
-    {% endif %}
+    <link rel='preload' href='/home-assistant-polymer/build/core.js' as='script'/>
```

Comment:
Preload only when not in dev mode?

Why is crossorigin needed - this is a local resource.

Reply:
Fonts need to be preloaded as crossorigin or else they are not picked up by the CSS and are then loaded twice.